src/dune_pkg/opam_solver.ml

```diff
@@ -386,17 +386,6 @@ let opam_commands_to_actions package (commands : OpamTypes.command list) =
     | [] -> None)
 ;;
 
-(* returns:
-   [None] if the command list is empty
-   [Some (Action.Run ...)] if there is a single command
-   [Some (Action.Progn [Action.Run ...; ...])] if there are multiple commands *)
-let opam_commands_to_action package (commands : OpamTypes.command list) =
```

Comment:
You could keep this function and not have to do the pattern match twice.

Reply:
Well not really since this matches on `opam_commands_to_actions`. That's why I introduced `prog_opt` before. If you like I can bring that back and use it instead.